packages/now-client/src/utils/index.ts

```diff
@@ -109,11 +109,16 @@ export async function getVercelIgnore(
 
   const files = await Promise.all(
     cwds.map(async cwd => {
-      let str = await maybeRead(join(cwd, '.vercelignore'), '');
-      if (!str) {
-        str = await maybeRead(join(cwd, '.nowignore'), '');
+      const [vercelignore, nowignore] = await Promise.all([
+        maybeRead(join(cwd, '.vercelignore'), ''),
+        maybeRead(join(cwd, '.nowignore'), ''),
+      ]);
+      if (vercelignore && nowignore) {
+        throw new Error(
```

Comment:
I think this is supposed to yield the error event instead of throw

Reply:
As in `return`? This function isn't an async generator…

Reply:
Oh I was thinking now-client uses an async generator. Maybe this is fine then.

Reply:
Ya, good point.